test/functional-portable/kubernetes/noncloud/deploymenttemplate_test.go

```diff
@@ -73,6 +73,15 @@ func Test_DeploymentTemplate_Env(t *testing.T) {
 	_, err = opts.K8sClient.CoreV1().Namespaces().Create(ctx, &corev1.Namespace{ObjectMeta: metav1.ObjectMeta{Name: namespace}}, metav1.CreateOptions{})
 	require.NoError(t, controller_runtime.IgnoreAlreadyExists(err))
 
+	// Clean up the namespace after the test
+	defer func() {
```

Comment:
The five deferred cleanup functions in this type are almost exactly the same. Can they be consolidated into a single function?